public/scripts/tagsList.js

```diff
@@ -1,21 +1,148 @@
-function getTagObject(content){
+let labelArray = null
+let currentFocus = -1
+
+function findLabelSuggestion(input) {
+    if (input === '')
+        return []
+    let suggestion = []
```

Comment:
Semi-colon after `let suggestion = [];`? Also, I'm not sure if a plain `"suggestion"` is the right variable name inside this context? Feels like you need to lend the `"autocomplete"` concept to rename the more appropriately. So maybe `"autocompleteSuggestion"`?